### ISSUE TITLE
JAWS doesn't read slider values when reading with virtual cursor

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/ohms-law#148]
From https://github.com/phetsims/QA/issues/327

@KatieWoe reported that JAWS exhibited https://github.com/phetsims/resistance-in-a-wire/issues/177 (which is a known JAWS bug, and we submitted a report). But this time it happened, JAWS failed to read the slider value. Here is a recording of it happening.

https://drive.google.com/file/d/116L6UpV235rqAKw6ILUWwsoO_X7LCbL4/view?usp=sharing

**Comment by jessegreenberg** [comment]
Assigning to myself to try to reproduce.

**Comment by jessegreenberg** [comment]
JSFiddle to test:  https://jsfiddle.net/z945gaf7/

**Comment by terracoda** [comment]
@jessegreenberg, thanks for finding this difference.
 Let's remove the list structure. I was thinking we should do that in GFLB's Mass controls, too.

Question is if we should group the two controls in a different way, like in a `div` with `aria-labelledby`.

We could remove the list, then test an alternate grouping for a maintenance release. What do you think?

**Comment by jessegreenberg** [comment]
OK, that sounds great @terracoda I will do that. I will remove the list for this release, then we can talk about another way to group for maintenance or the next release.

**Comment by jessegreenberg** [comment]
Done in the above commit, I tested that JAWS and NVDA read the value when reading with the virtual cursor. Changes cherry picked to 1.4 release branch
